### PR TITLE
Fetch data

### PR DESCRIPTION
**Pull request by hedita**

### CODE REVIEW

File: quotes.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+let quotes = document.getElementById("quote");
+let author = document.getElementById("author");
+let tags = document.getElementById("tag-item");
+
+async function getQuotes() {
+  let result = await fetch("https://api.quotable.io/random");
+  let resultJson = await result.json();
+
+  let content = resultJson.content;
+  let writer = resultJson.author;
```
Comment (blacksrc) [inline quotes.js]: This is totally fine, Just a simple tip. You can change this to:
```js
const {content , author} = resultJson;
```

Comment (blacksrc) [inline quotes.js]: You can change all `let` to `const`.

Comment (blacksrc) [reply quotes.js]: or if you want to go even further:
```js
const {content , author} = await result.json();
```

Comment (blacksrc) [reply quotes.js]: or even more (don't do that).
```js
const {content , author} = await (await fetch("https://api.quotable.io/random")).json();
```

Comment (hedita) [reply quotes.js]: Done.

File: quotes.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+const author = document.getElementById("author");
+const tags = document.getElementById("tag-item");
+
+async function getQuotes() {
+  const result = await fetch("https://api.quotable.io/random");
+  const resultJson = await result.json();
+
+  const content = resultJson.content;
+  const writer = resultJson.author;
+  const tagsArr = resultJson.tags;
```
Comment (blacksrc) [inline quotes.js]: can be:
```js
const { content, author, tags: tagsArray } = resultJson
```

File: quotes.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+  quotes.innerHTML = content;
+  author.innerHTML = writer;
+  tagsArr.forEach(tag => {
+    tags.innerHTML = tag;
+  });
```
Comment (blacksrc) [inline quotes.js]: if you have multiple tags in the response. It always shows the last tag.

Comment (blacksrc) [reply quotes.js]: you have to build the the whole `<li id="tag-item" class="tag-item"></li>` inside the loop and after the loop you should change the inner html of `ul`.

File: quotes.js (hunk 1 above)
Comment (hedita) [reply quotes.js]: Done.

File: quotes.js (hunk 2 above)
Comment (hedita) [reply quotes.js]: Done.

File: quotes.js [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+const tags = document.getElementById("tag-item");
+
+async function getQuotes() {
+  const result = await fetch("https://api.quotable.io/random");
+  const resultJson = await result.json();
+
+  const {content , writer , tags: tagsArray}  = resultJson;
+
+  quotes.innerHTML = resultJson.content;
+  author.innerHTML = resultJson.author;
```
Comment (blacksrc) [inline quotes.js]:
```suggestion
  const {content , author , tags: tagsArray}  = resultJson;

  quotes.innerHTML = content;
  author.innerHTML = author;
```

File: quotes.js [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+  const result = await fetch("https://api.quotable.io/random");
+  const resultJson = await result.json();
+  const {content , author , tags: tagsArray}  = resultJson;
+
+  quotes.innerHTML = content;
+  writer.innerHTML = author;
+  tagsArray.forEach(tag => {
+  listHtml += `<li class="item"> ${tag} </li>`
+  tagsList.innerHTML = listHtml;
+  });
```
Comment (blacksrc) [inline quotes.js]: indentation?

File: index.html [hunk 6]
```diff
@@ -9,15 +9,12 @@
...
-        created them.
-      </q>
-      <p class="author">Albert Einstein</p>
-      <ul class="tag-list">
-        <li class="tag-item">Wisdom</li>
-        <li class="tag-item">Famous-qoutes</li>
+      <q id="quote" class="quote"></q>
+      <p id="author" class="author"></p>
+      <ul id="tag-list" class="tag-list">
+        <li id="tag-item" class="tag-item"></li>
```
Comment (blacksrc) [inline index.html]: why we have empty li here?

File: quotes.js [hunk 7]
```diff
@@ -0,0 +1,21 @@
...
+async function getQuotes() {
+  const result = await fetch("https://api.quotable.io/random");
+  const resultJson = await result.json();
+  const {content , author , tags: tagsArray}  = resultJson;
+
+  quotes.innerHTML = content;
+  writer.innerHTML = author;
+  tagsArray.forEach(tag => {
+  listHtml += `<li class="item"> ${tag} </li>`
+  tagsList.innerHTML = listHtml;
```
Comment (blacksrc) [inline quotes.js]: should we move this line out of forEach or not? and why?

File: quotes.js (hunk 3 above)
Comment (hedita) [reply quotes.js]: Done.

File: index.html (hunk 6 above)
Comment (hedita) [reply index.html]: It was extra.I removed it.

File: quotes.js (hunk 5 above)
Comment (hedita) [reply quotes.js]: Done.

File: assets/css/structure.css [hunk 8]
```diff
@@ -22,7 +22,7 @@
   padding-top: var(--spacing-large);
 }
 
-.tag-item {
+.item {
```
Comment (blacksrc) [inline assets/css/structure.css]: why?

File: quotes.js (hunk 5 above)
Comment (blacksrc) [reply quotes.js]: it's not done.

File: assets/css/structure.css (hunk 8 above)
Comment (hedita) [reply assets/css/structure.css]: Done.

File: quotes.js [hunk 9]
```diff
@@ -0,0 +1,22 @@
...
+async function getQuotes() {
+  const result = await fetch("https://api.quotable.io/random");
+  const resultJson = await result.json();
+  const {content , author , tags: tagsArray}  = resultJson;
+
+  quotes.innerHTML = content;
+  writer.innerHTML = author;
+  tagsArray.forEach(tag => {
+  listHtml +=`<li class="tag-item">${tag}</li>`
+  //tagsList.innerHTML = listHtml;
```
Comment (blacksrc) [inline quotes.js]: why we have this comment?

File: quotes.js (hunk 5 above)
Comment (hedita) [reply quotes.js]: Fixed.

File: quotes.js (hunk 9 above)
Comment (hedita) [reply quotes.js]: Fixed.

File: quotes.js (hunk 7 above)
Comment (hedita) [reply quotes.js]: Yeah we should,actually it is unnecessary to write it in the loop.

**Comment by blacksrc** [comment]
:heavy_check_mark: LGTM

**Pull request merged**